metadata_backend/helpers/validator.py

```diff
@@ -23,7 +24,7 @@ def __init__(self, schema: XMLSchema, xml: str) -> None:
         """Set variables.
 
         :param schema: Schema to be used
-        :param content: Content of XML file to be validated
+        :param xml: Content of XML file to be validated
```

Comment:
Nice thing that code style checks miss these things... :D

Reply:
Yea, it's `pylint` that checks these things. I'm trying to configure it :)